### ISSUE TITLE
span with data-f-bind: nothing appears if value of variable is false

### ISSUE THREAD

**Issue by mmrj** [forio/flow.js#29]
in the following code, `sampleBool` is `true` and `sampleFalseBool` is `false`.
the span with 'sampleFalseBool' does NOT appear (the input, however, does), using Flow.js 0.8.3:

```
 sampleBool is currently (span and input below):
            <span data-f-bind="sampleBool"></span>
            <input data-f-bind="sampleBool"></input>
<br>
confirm: sampleFalseBool is currently (span and input below):
            <span data-f-bind="sampleFalseBool"></span>
            <input data-f-bind="sampleFalseBool"></input>
```

expected behavior is to see "false" output where the span is.

**Comment by narenranjit** [comment]
fixed in edge/ upcoming 0.9

**Comment by mmrj** [comment]
verified in 0.9.0
